src/Kris/LaravelFormBuilder/Form.php

```diff
@@ -618,6 +630,19 @@ public function __get($name)
     }
 
     /**
+     * Set the Event Dispatcher to fire Laravel events
+     *
+     * @param EventDispatcher $eventDispatcher
+     * @return $this
+     */
+    public function setEventDispatcher(EventDispatcher $eventDispatcher)
```

Comment:
Ah, my bad. Thought you are passing it to FormHelper through constructor. You can leave it like this.